components/operators/OperatorSummary.tsx

```diff
@@ -87,7 +93,7 @@ export const OperatorSummary = forwardRef<HTMLButtonElement, OperatorSummaryProp
                   <IconSwitch
                     on={isFavorite}
                     size='large'
-                    icon={<FontAwesomeIcon icon={faStar} />}
+                    icon={<FontAwesomeIcon icon={isFavorite ? faStarSolid : faStarLight} />}
```

Comment:
We should use both `offIcon` and `onIcon` parameters, passing respectively `faStarLight` and `faStarSolid`